server/commands/subscriptionCreator.ts

```diff
@@ -36,8 +36,23 @@ export default async function subscriptionCreator({
     paranoid: false,
   });
 
-  // Don't emit an event if a new subscription
-  // wasn't created.
+  // Fetched an already deleted subscription.
+  if (subscription.deletedAt) {
+    subscription.update({ deletedAt: null });
+
+    await Event.create(
+      {
+        name: "subscriptions.create",
+        modelId: subscription.id,
+        actorId: user.id,
+        userId: user.id,
```

Comment:
```suggestion
```

`userId` is for events that affect a user.

Reply:
Gotcha! This applies everywhere subscription might emit an event, would that be correct? 

I am assuming `actorId` should be the _indicator_ of user's identity.

Reply:
Actually, I'm wrong sorry leave `userId` here as it's on the Subscription model. apologies

Reply:
Understood, no worries :+1: